src/v/cluster/archival/ntp_archiver_service.cc

```diff
@@ -382,6 +390,87 @@ ntp_archiver::ntp_archiver(
     }
 }
 
+void ntp_archiver::log_collected_traces() noexcept {
+    try {
+        _rtclog.bypass_tracing([this] {
+            _rtclog.info("Diagnostic dump start");
```

Comment:
What about logging at error level so that this fails our CI tests? Or, shall we update ducktape regexp to catch these?